openedx/core/djangoapps/api_admin/utils.py

```diff
@@ -11,5 +13,5 @@ def course_discovery_api_client(user):
     course_discovery_client = Client.objects.get(name=CLIENT_NAME)
```

Comment:
It might be better to create a new method that doesn't rely on `Client` objects. Long-term we should not be relying on them.